### DISCUSSION TITLE
Ctrl+Tab?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#16521] [General]
Does Ctrl+Tab do anything?  It's not sent to my shell?

Is it possible to search the issues/discussions for "Ctrl+Tab"?

**Comment by zadjii-msft** [comment]
In conhost? Terminal? 

Do you have it bound in your settings? Do note, they're bound in the Terminal's default settings: https://github.com/microsoft/terminal/blob/63c3573a13cc96bfa3d655f0f1898ae250e2fe35/src/cascadia/TerminalSettingsModel/defaults.json#L401-L402

**Comment by vefatica** [comment]
I don't set it specifically.  I found it in Settings\Actions.  It is not mentioned in the Command Palette.  So I'm OK.
